src/transaction/log_page_buffer.c

```diff
@@ -2482,6 +2482,13 @@ logpb_fetch_start_append_page (THREAD_ENTRY * thread_p)
 
   logpb_log ("started logpb_fetch_start_append_page\n");
 
+  if (prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID) != 0
+      && (LOG_PAGEID) prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID) == log_Gl.hdr.append_lsa.pageid
+      && log_Gl.hdr.append_lsa.offset == 0)
+    {
+      flag = NEW_PAGE;
+    }
+
```

Comment:
```suggestion
  if ((LOG_PAGEID) prm_get_bigint_value (PRM_ID_FIRST_LOG_PAGEID) == log_Gl.hdr.append_lsa.pageid
      && log_Gl.hdr.append_lsa.offset == 0)
    {
      flag = NEW_PAGE;
    }

```

It may remove the if conditions at line 2493 ~ 2496 
but if you intended to distinguish this exceptional case from original case, I think it is OK.

Reply:
@joohok You are right.